packages/edit-post/src/components/layout/index.js

```diff
@@ -124,7 +145,53 @@ function Layout() {
 			<FocusReturnProvider>
 				<InterfaceSkeleton
 					className={ className }
-					header={ <Header /> }
+					header={
+						<Header
+							isInserterOpen={ isInserterOpen }
+							onToggleInserter={ () =>
+								setIsInserterOpen( ! isInserterOpen )
+							}
+						/>
+					}
+					leftSidebar={
+						mode === 'visual' &&
+						isInserterOpen && (
+							<Animate
```

Comment:
I think the animation is a bit off. Can we remove it a add it back with springs?

Reply:
What's wrong with the animation, it's the same one used in the publish panel.

Reply:
The easing. Neither seems refined enough.